WinRT.Runtime/Projections/INotifyCollectionChanged.cs

```diff
@@ -52,14 +52,29 @@ static unsafe Vftbl()
             }
 
             private static global::System.Runtime.CompilerServices.ConditionalWeakTable<global::System.Collections.Specialized.INotifyCollectionChanged, global::WinRT.EventRegistrationTokenTable<global::System.Collections.Specialized.NotifyCollectionChangedEventHandler>> _CollectionChanged_TokenTables = new global::System.Runtime.CompilerServices.ConditionalWeakTable<global::System.Collections.Specialized.INotifyCollectionChanged, global::WinRT.EventRegistrationTokenTable<global::System.Collections.Specialized.NotifyCollectionChangedEventHandler>>();
+
+
+#if NETSTANDARD2_0
             private static unsafe int Do_Abi_add_CollectionChanged_0(IntPtr thisPtr, IntPtr handler, out global::WinRT.EventRegistrationToken token)
             {
-                token = default;
+                fixed (global::WinRT.EventRegistrationToken* ptoken = &token)
+                {
+                    return Do_Abi_add_CollectionChanged_0(thisPtr, handler, ptoken);
+                }
+            }
+#endif
+
+#if !NETSTANDARD2_0
+            [UnmanagedCallersOnly]
```

Comment:
our code-gen now puts this out:
	            [UnmanagedCallersOnly(CallConvs = new [] { typeof(CallConvStdcall) })]

any concern with the custom mappings not doing likewise?

Reply:
Nope. The manual type mappings have either been updated to handle this correctly, use the Unmanaged calling convention on .NET 5 (which maps to stdcall on Windows), or are generic and as a result still use delegates.